src/components/Highlights.vue

```diff
@@ -1,26 +1,181 @@
 <template>
   <div class="highlights-bar">
-    <div class="betas">JSE Index:{{ jseIndex }}</div>
-    <div class="betas">{{ industryIndices[1] }}: {{ industryBetas[1] }}</div>
-    <div class="betas">{{ industryIndices[2] }}: {{ industryBetas[2] }}</div>
+    <div class="betas">
+      All Share Index (30 Days): {{ index1 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
+          positive: index1Change >= 0,
+          negative: !(index1Change >= 0),
+        }"
+        v-on:mount="index1Sign = index1Change <= 0"
+      >
+        ({{ index1Change }}%)
+      </span>
+    </div>
+
+    <div class="betas">
+      TOP 40 Index (30 Days): {{ index2 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
+          positive: index2Change >= 0,
+          negative: !(index2Change >= 0),
+        }"
+        v-on:mount="index1Sign = index2Change <= 0"
+      >
+        ({{ index2Change }}%)
+      </span>
+    </div>
+
+    <div class="betas">
+      F&I Index (30 Days): {{ index3 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
+          positive: index3Change >= 0,
+          negative: !(index3Change >= 0),
+        }"
+        v-on:mount="index3Sign = index3Change <= 0"
+      >
+        ({{ index3Change }}%)
+      </span>
+    </div>
+
+    <div class="betas">
+      Industrials Index (30 Days): {{ index4 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
+          positive: index4Change >= 0,
+          negative: !(index4Change >= 0),
+        }"
+        v-on:mount="index4Sign = index4Change <= 0"
+      >
+        ({{ index4Change }}%)
+      </span>
+    </div>
+
+    <div class="betas">
+      Resources Index (30 Days): {{ index5 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
+          positive: index5Change >= 0,
+          negative: !(index5Change >= 0),
+        }"
+        v-on:mount="index5Sign = index5Change <= 0"
+      >
+        ({{ index5Change }}%)
+      </span>
+    </div>
   </div>
 </template>
 
 <script>
+//Axios stuff
```

Comment:
Is this comment necessary?

Reply:
I can take it out

Reply:
@RationalRyan11 unlike what UCT lecturers seem to think, comments should be avoided at all cost unless you are describing a hack that needed to be done for some reason (which is like never). Your code should explain what it by have clear variable and function names so that when I read through it I know exactly what is going on.

A good sanity test is if you ever feel like you need to comment, then you need to refactor your code to be clearer so the comment isn't necessary.